.github/workflows/build_wheels.yml

```diff
@@ -256,7 +256,7 @@ jobs:
           echo "Tagging ${BRANCH_NAME}"
           git tag -f nightly-${BRANCH_NAME} HEAD
       - name: Push tags
-        uses: apache/airflow-github-push-action@b007e7b818e33b04afd056e4c4b57ba917145d7a
+        uses: ./.github/actions/github-push-action
```

Comment:
You need to checkout the submodules on the checkout action just above this. Look at the Airflow ci.yml workflow.

(Sorry, on phone else I'd be more precise)